.github/workflows/claude.yml

```
@@ -10,6 +10,14 @@ on:
   pull_request_review:
     types: [submitted]
 
+# Queue rather than cancel: cancelling mid-run leaves a half-applied edit on the branch.
+# `queue: max` keeps every pending request; the default (`single`) would cancel an
+# already-pending run when a second @claude comment arrives, silently dropping it.
+concurrency:
+  group: claude-${{ github.event.issue.number || github.event.pull_request.number }}
+  cancel-in-progress: false
+  queue: max
```

actionlint (1.7.12)

[error] unexpected key "queue" for "concurrency" section. expected one of "cancel-in-progress", "group"

(syntax-check)